draft-ietf-httpbis-messaging-latest.xml

```diff
@@ -2024,7 +2024,7 @@ https://www.example.org
 </t>
 </section>
 
-<section title="Upgrade" anchor="header.upgrade">
+<section title="Upgrading Connections" anchor="header.upgrade">
```

Comment:
I would either leave the title as is, or insert a subsection that defines the field itself.